_data/issues/date-picker.yml

```
@@ -1,18 +1,13 @@
 title: Date picker
-type: component
-alert-type: info
+alert_type: info
 items:
 - date: 2023-09-29
   summary: Users had trouble correctly typing dates in the input field.
-  summaryAdditional:
-  affectsAll: true
-  affectsScreenReaders: true
-  githubRepo: uswds
-  githubNumber: 5534
+  summary_additional:
+  issue_repo: uswds
+  issue_number: 5534
 - date: 2023-09-29
   summary: The focus indicator is not visible around the calendar icon in high contrast mode.
```

I was wondering if there was a plainer way to say it, but if "focus indicator" is a common phrase or term we use, we should leave it.

---

```suggestion
  summary: In high contrast mode, the calendar icon does not display distinct focus styles.
```
What about something like this? 
I also wanted to note that a fix for this issue is slated for 3.8.0 release, so if all goes to plan this item will not be on the page long (if at all). So it probably isn't crucial to spend a lot of time re-working this.

---

Resolving this comment because this item was removed in preparation of the 3.8.0 release.